<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditFlight</class>
 <widget class="QWidget" name="EditFlight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="datetime_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Дата (в формате &quot;dd.mm.yyyy hh:mm&quot;)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="datetime_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="way_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Редактировать историю вылетов</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="id_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>id летательного средства</string>
   </property>
  </widget>
  <widget class="QLabel" name="way_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Рейс</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить запись</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
